packages/amplication-server/src/core/build/build.service.ts

```diff
@@ -314,7 +324,7 @@ export class BuildService {
     build: Build,
```

Comment:
@EugeneTseitlin please remove build from this function this is redundant - remove it from the create function as well

Reply:
done